internal/provider/resource_access_list_test.go

```diff
@@ -27,7 +26,15 @@ resource "astra_access_list" "example" {
   database_id = "aba3cf20-d579-4091-a36d-9c9f75096031"
   addresses {
     request {
-      address= "0.0.0.0/0"
+      address= "0.0.0.1/0"
```

Comment:
Is it worth some additional coverage to add a `not enabled` to the test mix?

Reply:
I could, but to be honest, these tests just do an ADD, then a READ, then a DELETE and make sure none of those operations fail. Also, the READ doesn't verify that what you added got added, nor does the DELETE verify that everything got removed. Testing in general needs some more attention....

Reply:
💯